inc/Editor/DataBinding/BlockBindings.php

```diff
@@ -204,18 +204,24 @@ private static function execute_queries( array $block_context, array $source_arg
 		}
 	}
 
-	public static function should_render_empty_result( WP_Block $block ): bool {
-		$block_context = $block->context[ self::$context_name ] ?? [];
+	public static function is_error_or_empty_state( array $context, array $attributes ): bool {
```

Comment:
I think I might prefer something like `should_render_fallback_content` since that is its intended purpose.

Reply:
Happy to go with this method name too, instead of `should_render_no_result_state`